components/avatar.css

```diff
@@ -0,0 +1,20 @@
+.avatar{
+    border-radius: 50%;
+    margin:0 2px;
+}
+.avatar-lg{
+    height: 150px;
+    width: 150px;
+}
+.avatar-md{
+    height: 100px;
+    width: 100px;
+}
+.avatar-sm{
+    height: 50px;
+    width: 50px;
+}
+.avatar-xs{
+    height: 25px;
+    width: 25px;
```

Comment:
Use `rem` instead of `px`. It will help you a lot in making your apps responsive.

Reply:
Ok, I will use this. Thanks for reminding me.

Reply:
hey, I fixed that can you please approve this?